src/Cli/StartupJokeSpeechBubble.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buggregator\Trap\Cli;
+
+class StartupJokeSpeechBubble
+{
+    public const JOKES = [
+        'Why do programmers always mix up Halloween and Christmas? Because Oct 31 == Dec 25.',
+        'Two hard things in computer science: cache invalidation, naming things and stack overflow.',
+        'Depressive programming style through dump and die.',
+        'PHP was dead 84 years ago right?',
+    ];
+
+    public static function randomStartupJokeSpeechBubble(): string
+    {
+        $joke = self::JOKES[mt_rand(0, \count(self::JOKES) - 1)];
```

Comment:
```suggestion
        $joke = self::JOKES[\mt_rand(0, \count(self::JOKES) - 1)];
```

It's preffered to use FQN for functions from global scope